modules/basic-skills/src/actions/choice_invalid_answer.js

```diff
@@ -2,6 +2,6 @@
  * @hidden true
  */
 
-const key = 'skill-choice-invalid-count'
+const key = args.randomId ? `skill-choice-invalid-count-${args.randomId}` : `skill-choice-invalid-count`
```

Comment:
You don't need the ternary skill-choice-invalid-count-${args.randomId} with an undefined random id will always render the same key which keeps the wanted behavior

Reply:
Nop, the issue is that existing skills uses 'skill-choice-invalid-count', and if we leave it like this, it will use 'skill-choice-invalid-count-undefined' which will break existing ones